pages/common/dlv.md

```diff
@@ -0,0 +1,24 @@
+# dlv
+
+> Delve is a debugger for the Go programming language.
+> More information: <https://github.com/go-delve/delve/blob/master/Documentation/usage/dlv.md/>.
+
+- Compile and begin debugging main package in current directory, or the package specified:
+
+`dlv debug`
+
+- Compile test binary and begin debugging program:
+
+`dlv test`
+
+- Connect to a headless debug server:
+
+`dlv connect`
+
+- Attach to running process and begin debugging:
+
+`dlv attach`
```

Comment:
The docs say this needs a `pid`.

Reply:
Thanks for reviewing. Would this work?

'div attach {{pid}}'

Reply:
Yes.

Reply:
@pixelcmtd @navarroaxel Thanks, I made the changes. Review left on last line.